File: HorseRacing.Contracts/Models/Game/Responses/GetGameResponse.cs
using HorseRacing.Contracts.Base.Responses;
using HorseRacing.Contracts.Models.Game.Dtos;

namespace HorseRacing.Contracts.Models.Game.Responses
{
    public class GetGameResponse : BaseResponse<GetGameResponseDto>
    {
        public GetGameResponse() : base() { }

        public GetGameResponse(GetGameResponseDto data) : base(data) { }
    }
}

File: HorseRacing.Web/src/interfaces/api/contracts/model/game/responses/get-game/get-game-response.ts
import type { BaseResponse } from '../../../../base/responses/base-response';
import type { GetGameResponseDto } from './get-game-response-dto';

export class GetGameResponse implements BaseResponse<GetGameResponseDto> {
    Data: GetGameResponseDto;

    constructor(data?: GetGameResponseDto) {
        this.Data = data || {
            GameId: '',
            Name: '',
            Status: 0,
            Mode: 0,
            DefaultBet: 0,
            DateStart: '',
            DateEnd: '',
        };
    }
}
